pipelines/livepatching/DeployLivepatch.yml

```diff
@@ -86,28 +110,155 @@ extends:
             tags: '$(downloadTags)'
             artifact: $(buildArtifactsFolder)
             patterns: |
-              **/toolkit-*.tar.gz
-              **/toolchain_built_rpms_all.tar.gz
               **/rpms.tar.gz
+              **/toolchain_built_rpms_all.tar.gz
+              **/toolkit-*.tar.gz
             path: '$(System.ArtifactsDirectory)'
 
         - task: Bash@3
           displayName: 'Build livepatch'
           inputs:
-            filePath: '$(Build.SourcesDirectory)/pipelines/livepatching/BuildLivepatch.sh'
+            filePath: 'pipelines/livepatching/BuildLivepatch.sh'
             arguments: '-a "$(System.ArtifactsDirectory)"
                         -k "${{ parameters.livepatchKernelVersion }}"
                         -l "$(Build.ArtifactStagingDirectory)/$(buildLogsFolder)"
-                        -p "$(Build.ArtifactStagingDirectory)/$(buildArtifactsFolder)"
+                        -o "$(Build.ArtifactStagingDirectory)/$(buildArtifactsFolder)"
                         -s "${{ parameters.useRPMsSnapshot }}"'
 
     - stage: Sign
       jobs:
-      - job:
+      - job: Kernel_modules
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish build artifacts'
+            targetPath: $(Build.ArtifactStagingDirectory)/$(buildArtifactsFolder)
+            artifactName: $(buildArtifactsFolder)_KERNEL_MODULES
+
+        pool:
+          # The kernel modules signing task works only on Ubuntu.
+          name: mariner-core-x64-1es-ubuntu2204-gpt-test
```

Comment:
Just curious - why this pool instead of mariner-core-x64-1es-ubuntu-22.04 ?

Reply:
I'm using it for testing until the whole pipeline is finished (testing + publishing). I don't want to mess with the production pools in the meantime. After I'm done, this pool is going away and I'll switch to the one you mentioned. In case any adjustments are needed beforehand, I'll be sure to test them first.